app/main.py

```diff
@@ -1,13 +1,17 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+    return {["name", "message", "column", "line", "source"][i]
+            : error[["code", "text", "column_number", "line_number"][i]]
+            for i in range(4)} | {"source": "flake8"}
```

Comment:
So hard to read this part....
Make it as just a dict and pass all values manually or make it as:
```
{
    key: error[error_key]
    for key, error_key in (
        ("name", "code"),
        ("message", "text"),
        ("line", "column_number"),
        ("source", "line_number"),
    )
}

```
But manually looks more better